devtools/platform-descriptor-json-plugin/src/main/java/io/quarkus/maven/GenerateExtensionsJsonMojo.java

```diff
@@ -335,6 +335,12 @@ private String extensionId(JsonObject extObject) {
 
     private JsonObject mergeObject(JsonObject extObject, JsonObject extOverride) {
         final JsonObjectBuilder mergedObject = Json.createObjectBuilder();
```

Comment:
What I think is that something went missing in this line. I think this should be `Json.createObjectBuilder(extObject);` or something like that (didn't test yet)

Reply:
that could probably work too!